docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -2125,20 +2125,21 @@ Result:
 
 ### getRecentPrioritizationFees
 
+**UNSTABLE**
```

Comment:
Practically I don't how flagging this method as unstable matters. Once clients adopt it, and we really hope they will quickly, it's going to be very tough to make non-backwards compatible changes to it.

Reply:
Hmm, yeah, that's fair. I am hoping any future changes are additive regardless.